src/concepts/segments.md

```diff
@@ -67,11 +67,40 @@ User states are found as columns in the `clients_last_seen` dataset: the user st
 
 ### WAU and MAU
 
-Users can change user states part way through a week or a month. 
-At present, 
-we have not settled the questions of whether and how to split MAU by user state,
-and whether and how to measure MAU for each user state.
-So stick to DAU for now.
+Users can move in or out of specific user states part way through a week or a month.
+This poses a conundrum if we want to plot the WAU or MAU for a user state.
+
+Our convention is to _count the number of distinct users who were active in user state in the period_.
+So if a user was active as a regular user v3 on e.g. the second day of a 28-day window, then they will contribute to "regular user v3 MAU" regardless of whether they lost their "regular user v3" status at any point in the 28-day window.
+
+Since many user states use the full extent of the `*_bits` column wizardry in `clients_last_seen`, you'll have to query WAU or MAU the old fashioned way:
+
+```lang=sql
+WITH dates AS (
+    SELECT *
+    FROM UNNEST(GENERATE_DATE_ARRAY('2020-05-01', '2020-07-01')) as d
+) SELECT
+    dates.d AS submission_date,
+    COUNT(DISTINCT client_id) * 100 AS regular_user_v3_mau,
+FROM dates
+INNER JOIN telemetry.clients_last_seen cls
+    ON cls.submission_date BETWEEN DATE_SUB(dates.d, INTERVAL 27 DAY) AND dates.d
+    AND cls.submission_date BETWEEN '2020-04-01' AND '2020-07-01'
+WHERE cls.sample_id = 42
+    AND cls.is_regular_user_v3
+    AND cls.days_since_seen = 0
+GROUP BY dates.d
+ORDER BY dates.d
+```
+
+Our convention has the potentially counterintuitive consequence that a user can count towards "regular user v3 MAU" and "not regular user v3 MAU" for the same 28-day window.
+If you need to break MAU down into the sum of MAU for various user states, then in this instance you would need to break it down into "regular v3", "not regular v3", and "transitioning".
```

Comment:
It seems like transitioning essentially means "qualified as both regular and not regular in the same period but at different times". 

This is a nit, but to me, transitioning implies a one-time shift either into or out of a state, where in this case it could be a lot messier, e.g. dropping in and then out again multiple times. But I'm not sure I have a better term for "transitioning" other than "both regular and not-regular at different times". "Irregularly regular" is just awful. Maybe not-regular, full-time-regular, and part-time-regular?

Reply:
Yeah, good call. "both" will do in the context of this sentence. It might not be a great option in practice - i.e. out of context on a graph legend - but it gets the point across for this document. So let's leave the search for a "great option in practice" as an exercise for the reader :grinning: